lib/reporting/protocols_report.rb

```diff
@@ -318,14 +309,64 @@ def aal3_issuers_query
         | parse @message '"authn_context":[*]' as authn
         | filter
           name IN %{event}
-          AND (authn like /aal\\/3/ or acr like /aal\\/3/)
+          AND (authn like /ns\\/assurance\\/loa/ OR acr like /ns\\/assurance\\/loa/)
           AND properties.event_properties.success= 1
         | display issuer
         | sort issuer
         | dedup issuer
       QUERY
     end
 
+    def protocol_query
+      params = {
+        event: quote([SAML_AUTH_EVENT, OIDC_AUTH_EVENT]),
+      }
+
+      format(<<~QUERY, params)
+        fields
+          name AS protocol,
+          coalesce(properties.event_properties.service_provider, properties.event_properties.client_id) as issuer
+        | filter name IN %{event} AND properties.event_properties.success= 1
+        | stats
+            count(*) AS request_count
+          BY
+          protocol, issuer
+      QUERY
+    end
+
+    def saml_signature_query
+      params = {
+        event: quote([SAML_AUTH_EVENT]),
+      }
+
+      format(<<~QUERY, params)
+        fields
+          properties.event_properties.service_provider AS issuer,
+          properties.event_properties.request_signed = 1 AS signed,
+          properties.event_properties.request_signed != 1 AS not_signed,
+          isempty(properties.event_properties.matching_cert_serial) AND signed AS invalid_signature
+        | filter name IN %{event}
```

Comment:
events (plural)
```suggestion
        events: quote([SAML_AUTH_EVENT]),
      }

      format(<<~QUERY, params)
        fields
          properties.event_properties.service_provider AS issuer,
          properties.event_properties.request_signed = 1 AS signed,
          properties.event_properties.request_signed != 1 AS not_signed,
          isempty(properties.event_properties.matching_cert_serial) AND signed AS invalid_signature
        | filter name IN %{events}
```

Reply:
fixed in https://github.com/18F/identity-idp/pull/11339/commits/e5d168559ef6cf72aa260b3046d1bbf82be121a4